sky/engine/core/script/dart_init.cc

```diff
@@ -307,6 +308,61 @@ bool IsRunningPrecompiledCode() {
 
 #endif  // DART_ALLOW_DYNAMIC_RESOLUTION
 
+static base::LazyInstance<std::unique_ptr<EmbedderTracingCallbacks>>::Leaky
```

Comment:
I couldn't make `g_tracing_callbacks` a unique ptr because that would require an exit-time destructor (we check via `-Wexit-time-destructors`). And using raw pointers seemed icky when I had to get rid of the previous callbacks instance in `SetEmbedderTracingCallbacks`. We dont actually leak the callbacks since the tracing controller cleans up its callbacks when it goes away.